scripts/mail-alarm

```diff
@@ -502,11 +502,19 @@ class XapiMessage:
             self.body      = ""
         self.pool_name = get_pool_name()
         try:
-            with open('/etc/xapi.d/mail-languages/' + mail_language + '.json', 'r') as fileh:
-                self.ml = json.loads(fileh.read())
+            mail_language_file = '/etc/xapi.d/mail-languages/' + mail_language + '.json'
+            with open(mail_language_file, 'r') as fileh:
+                file_content = fileh.read()
+                self.mail_language = json.loads(file_content, encoding="utf-8")
```

Comment:
But we are not printing the content, and in case we really need it (and for some reason we cannot access the file) we can always make the change temporarily in a dev rpm.